src/_includes/partials/search.njk

```diff
@@ -0,0 +1,5 @@
+{% if not search %}
+<gcds-search search-id="search" slot="search" placeholder="GC Design System" action="/en/search">
```

Comment:
I think this needs to know the language, otherwise the french site goes to `/en/search`
```suggestion
<gcds-search search-id="search" slot="search" placeholder="GC Design System" action="/en/search">
```